File: zerojudge/c067/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    long long int n, hi, a, c,set = 1;
    vector <int> vec;
    while (cin>>n){
        if (n == 0){
            break;
        }
        vec.clear();
        c = 0;
        a = 0;
        for (int t=0; t<n; t++){
            cin>>hi;
            vec.push_back(hi);
            a += hi;
        }
        a /= n;
        for (int t=0; t < vec.size(); t++){
            if (vec[t] > a){
                c += vec[t] - a;
            }
        }
        cout<<"Set #"<<set<<endl;
        cout<<"The minimum number of moves is "<<c<<"."<<endl;
        cout<<endl;
        set ++;
    }
    return 0;
}

File: zerojudge/c067/main.py
Set = 1
while True:
    n = int(input())
    if n == 0:
        break
    item = []
    inp_text = input()
    inp_text = inp_text.split(" ")
    for t in range(n):
        item.append(int(inp_text[t]))
    avg = sum(item)/n
    move = 0
    for t in range(len(item)):
        if item[t] > avg:
            move += item[t] - avg
    print(f"Set #{Set}")
    print(f"The minimum number of moves is {int(move)}.\n")
    Set += 1